package day42_Exceptions_Throw_ThrowsKeywords;

public class ThrowsKeyword2 {

    public static void main(String[] args) throws ArithmeticException{

        System.out.println("----------------Test started-----------------");

        System.out.println(8/0);

        System.out.println("----------------Test ended--------------------");



    }
}
